src/sql/foglamp_ddl.sql

```diff
@@ -234,6 +233,15 @@ CREATE SEQUENCE foglamp.users_id_seq
     CACHE 1;
 ALTER SEQUENCE foglamp.users_id_seq OWNER TO foglamp;
 
+CREATE SEQUENCE foglamp.backups_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
```

Comment:
Is the change in DDL approved by @MarkRiddoch and @izoratti ? MAXVALUE 9223372036854775807 ? Do we expect these many number of backups on a constrained device?